distributions/openhab/src/main/resources/runtime/etc/jetty.xml

```diff
@@ -79,10 +79,10 @@
 		</Arg>
 	</Call>
 
-	<New id="httpConfig" class="org.eclipse.jetty.server.HttpConfiguration">
+	<New id="http2Config" class="org.eclipse.jetty.server.HttpConfiguration">
 		<Set name="secureScheme">https</Set>
 		<Set name="securePort">
-			<Property name="org.osgi.service.http.port.secure" default="8443" />
```

Comment:
Don't change that when you remove http/1.1 support, otherwise the setting of the port from the usual openhab-configuration will not work.

Reply:
Sure - I changed the other occurences back but did not notice this one here.